custom_components/tuya_local/devices/kogan_KASMCDSKTLA_kettle.yaml

```diff
@@ -0,0 +1,65 @@
+name: Kogan 1.7L Double Wall Glass Smart Kettle
+products:
+  - id: 7occ66lr74vfznwt
+    manufacturer: Kogan
+    model: KASMCDSKTLA
+entities:
+  - entity: switch
+    name: Power
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+  - entity: sensor
+    name: Current temperature
+    class: temperature
+    dps:
+      - id: 2
+        name: sensor
+        type: integer
+        unit: °C
+  - entity: number
+    name: Target Temperature
+    category: config
+    dps:
+      - id: 4
+        name: value
+        type: integer
+        unit: °C
+        range:
+          min: 40
+          max: 100
+  - entity: select
+    name: Status
+    category: diagnostic
+    dps:
+      - id: 8
+        name: option
+        type: string
+        mapping:
+          - dps_val: standby
+            value: Standby
+          - dps_val: heating
+            value: Heating
+          - dps_val: cooling
+            value: Cooling
+  - entity: select
+    name: Preset
```

Comment:
Use `translation_key: kettle_mode` instead of this name, and the options: `water`, `custom`, `coffee`, `green_tea`, `honey_water` and `infant_formula`